spec/diddocuments.md

```diff
@@ -184,8 +185,186 @@ Secp256k1 public keys must be converted to a verification method with a type of
 
 TODO: Add example of Secp256k1 key in a KEL and the resultant verification method
 
+##### Thresholds
+If the current signing keys threshold (the value of the `kt` field) is a string containing a number that is greater than 1,
+or if it is an array containing fractionally weighted thresholds, then in addition to the verification
+methods generated according to the rules in the previous sections, another verification method
+with a type of `ConditionalProof2022` will be generated in the DID document. This verification method type is defined here:
+https://w3c-ccg.github.io/verifiable-conditions/ (TODO: add proper references).
+
+It is constructed according to the following rules:
+
+* The `id` property of the verification method MUST be a relative DID URL and use the AID as the value of the fragment component, e.g., `"id": "#<aid>"`.
```

Comment:
I’m still a bit confused as to why this is `#<aid>` vs. `#<multibase of the public key>` (or some other identifier for the public key).  The value in the KERI event stream/state is a public key, and this is a reference to the `verificationMethod` in the DIDDoc for the public key. There is no information in the KERI data about the management of the public key.

Or maybe asked another way — how does one derive an AID from a “naked” public key+key type?

Reply:
I wasn't sure about this either. In a DID document, every verification method needs a fragment identifier. For single keys, I decided to use the CESR encoding of the public key itself as an identifier, see https://github.com/trustoverip/tswg-did-method-webs-specification/pull/39. This felt consistent with e.g. how the did:key method identifies keys in its DID document.

This PR here is about a "special" verification method that represents the threshold/multisig functionality of KERI. This verification method is not itself a key, but merely links to other keys. I wasn't sure what fragment identifier in a DID document would be appropriate for this, and I thought the AID itself, i.e. `#<aid>` was a good fit for identifying this "special" verification method. Another alternative could be to define some hardcoded identifier such as `#multisig` for this. I'm open to suggestions.

Reply:
OK — that’s what I thought.  I think I would use a different term then “<AID>”, as that implies (or it did to me) that the multi-sig keys are something more (and more complicated) than just public keys.  The `<AID>` is just an ID for a verificationMethod within the DIDDoc. I suggest just saying that, and the mechanism for generating the string from the public key can be defined separately.